src/retrace-server-bugzilla-query

```diff
@@ -47,37 +48,42 @@ if __name__ == "__main__":
         else:
             log.write("Not logged in. Continue as anonymous user.\n")
 
+        found_tasks: Dict[str, List[str]] = {}
         product_list = CONFIG.get_list("BugzillaProduct")
         component_list = CONFIG.get_list("BugzillaComponent")
-
-        query = bzapi.build_query(
-            product=product_list,
-            component=component_list,
-            include_fields=["id"])
-
-        # I don't want to run regex on each and every comment, at first try fast check
-        # if trigger word is in comment and if is, then use slower regex
+        config_status = CONFIG.get_list("BugzillaStatus")
         trigger_words = CONFIG.get_list("BugzillaTriggerWords")
         regexes = CONFIG.get_list("BugzillaRegExes")
+        if CONFIG["BugzillaQueryLastChangeDelta"]:
+            delta = CONFIG["BugzillaQueryLastChangeDelta"]
+        else:
+            delta = 7*24
```

Comment:
You don't need this check because of the way the `Config` class works. Just `CONFIG["BugzillaQueryLastChangeDelta"]` should be enough and it also keeps the defaults where they should be.

Reply:
Ok